src/index.js

```diff
@@ -7,6 +7,23 @@ import { StoreDetails } from "./components/StoreDetails";
 import { WonderCart } from "./components/WonderCart";
 import { createApiUrl } from "./sdk/createApiUrl";
 import { PluginsSdk } from "./sdk/plugins";
+import { HiiveAnalytics } from '@newfold-labs/js-utility-ui-analytics';
+import domReady from '@wordpress/dom-ready';
+
+domReady(() => {
+  HiiveAnalytics.initialize({
```

Comment:
Can we move the initialize function into the `sdk` as commented below

Reply:
kept as it is. as this method needs to initialise when the application starts.